src/problem.jl

```diff
@@ -19,15 +19,16 @@ JumpProblem(prob,aggregator::AbstractAggregatorAlgorithm,jumps::AbstractJump...;
 JumpProblem(prob,jumps::JumpSet;kwargs...) = JumpProblem(prob,NullAggregator(),jumps;kwargs...)
 
 function JumpProblem(prob,aggregator::AbstractAggregatorAlgorithm,jumps::JumpSet;
-                     save_positions = typeof(prob) <: AbstractDiscreteProblem ? (false,true) : (true,true))
+                     save_positions = typeof(prob) <: AbstractDiscreteProblem ? (false,true) : (true,true),
+                     rng = Xorshifts.Xoroshiro128Star(rand(UInt64)))
```

Comment:
Let's use Xoroshiro128Plus. See the end of the chat which states it's not the same as the one in the post.

Reply:
Just to clarify. The blog post found issues with Xoroshiro128Plus. It recommended as fine XorShift* 128/64. I haven't found any info about Xoroshiro128star.

Reply:
Oh really? But then it passed all of the tests of the repo?

Reply:
It failed a different series of tests (newer I think) called PractRand. I'm not an expert on these tests, so don't know the significance. You can see the details on the blog post I linked back in the issue thread.

Reply:
I don't know enough to comment on this but can change as needed.